src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1292,6 +1400,22 @@ inline void rpp_store8_f32_to_f32_avx(Rpp32f *dstPtr, __m256 *p)
     _mm256_storeu_ps(dstPtr, p[0]);
 }
 
+inline void rpp_load8_f32_to_f64_avx(Rpp32f *srcPtr, __m256d *p)
+{
+    __m128 px128[2];
+    __m256 px;
+    px = _mm256_loadu_ps(srcPtr);
+    px128[0] = _mm256_castps256_ps128(px);
+    px128[1] = _mm256_extractf128_ps(px, 1);
```

Comment:
Why have we used a cast for the 0 component and extract for the 1 component here?

Reply:
If its to save on time, lets check if there's any precision loss due to this